docs/site/development/updating-codes/update-language-script-info/language-script-description.md

```diff
@@ -4,19 +4,16 @@ title: Language Script Description
 
 # Language Script Description
 
-The language\_script spreadsheet should list all of the language / script combinations that are in common modern use. The countries are not important, since their function has been overtaken by the country\_language\_population spreadsheet.
+The [`language\_script.tsv`](https://github.com/unicode-org/cldr/blob/main/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language_script.tsv) data file should list all of the language / script combinations that are in common use. Usage by country is indicated in the [`country\_language\_population.tsv`](https://github.com/unicode-org/cldr/blob/main/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/country_language_population.tsv) spreadsheet.
 
-1. If the language and script are both modern, and the script is a major way to write the language in some country, then we should see that line marked as **primary**.
-2. Otherwise it should be marked **secondary**.
-
-Every language that is in official use in any country according to country\_language\_population  should have at least one primary script in the language\_script spreadsheet.
+1. Every language needs at least 1 script considered the **primary** script.
+    1. This data is used to determine [the most Likely language and region](likelysubtags-and-default-content) so there needs to be at least 1 primary value.
+    2. [Changed in v47] Include a primary script for historical languages (eg. Ancient Greek, Coptic). The primary script should reflect where the majority of the written corpus originates from.
+2. Languages written by significant populations with different scritps in different countries can have multiple **primary** scripts. The [likely subtags](https://www.unicode.org/cldr/charts/latest/supplemental/likely_subtags.html) patterns will use population counts to disambiguate the default script for each locale.
```

Comment:
I disagree with this; I think it complicates our work. It is simplest if there is exactly 1 primary script, period, that will match the maximized value for the language. All others are not needed, because they will be derived from language population data. Eg if Kurdish has Latin as the primary script, but ku_Arab is the largest population (of ku*) speakers, then we will generate ku_IQ => ku_Arab_IQ in likely subtags.

Is there any other purpose to multiple primaries or having secondaries?

Reply:
I'm fine with that to (only ever 1 primary). Do you think the CLDR TC agrees with that? I don't think I had much involvement.

I was trying to strike a compromise with the concerns people had about picking a primary language (that was mentioned in the DDL group). I'll bring it up again in the next DDL meeting.

Reply:
I agree that we should simply pick a primary, _just as_ we need to pick a single likely subtag (i.e. `zh-Hans`).  I know there are concerns, but structurally we just need to pick something.

Always having exactly 1 primary makes the most sense structurally.